frontend/src/features/public-form/components/FormFields/FormFields.tsx

```diff
@@ -27,18 +35,18 @@ export const FormFields = ({
   colorTheme,
   onSubmit,
 }: FormFieldsProps): JSX.Element => {
-  // TODO: Inject default values if field is also prefilled.
   const augmentedFormFields = useMemo(
-    () => formFields.map(augmentWithMyInfo),
+    () => formFields.map(augmentWithMyInfo).map(augmentWithMyInfoPrefill),
```

Comment:
do we need this here? public forms already have their fieldValue properly injected from the server. The TODO you removed is more for prefilled fields haha

Reply:
oh yea ps i forgot how the forms work - just to double check, we should just delete this cos the reducer below will inject the (actual) `prefilledValue` right?